apps/test/unit/lib/ui/SchoolInfoInterstitialTest.jsx

```diff
@@ -263,14 +263,8 @@ describe('SchoolInfoInterstitial', () => {
         />
       );
       wrapper.find(Button).simulate('click');
-      expect(server.requests[0].requestBody).to.equal(
-        [
-          '_method=patch',
-          'auth_token=fake_auth_token',
-          'user%5Bschool_info_attributes%5D%5Bcountry%5D=',
-          'user%5Bschool_info_attributes%5D%5Bschool_type%5D='
-        ].join('&')
-      );
+      expect(server.requests.length).to.equal(0);
+      expect(wrapper.state('errors').country).to.equal(true);
     });
 
     it('submits with only country=US', () => {
```

Comment:
The names of several of these tests need to be updated - they say it "submits" but the new behavior is that they don't submit.